lib/cb/requests/job_search/get.rb

```diff
@@ -18,6 +18,11 @@ def headers
             'Accept' => 'applicatin/json;version=3.0'
           }
         end
+        
+        def query
+          args.except(:three_scale_token)
```

Comment:
The parameters to the JobSearch API are in camel case.  The three_scale_token is a param set in the app.  There I went with snake case as that is our common way.  I actually didn't look at the way other params passed down are cased.